packages/flutter/lib/src/material/slider.dart

```diff
@@ -433,6 +474,27 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
     assert(debugCheckHasMaterial(context));
     assert(debugCheckHasMediaQuery(context));
 
+    switch (widget._sliderType) {
+      case _SliderType.material:
+        return _buildMaterialSlider(context);
+
+      case _SliderType.adaptive: {
+        final ThemeData theme = Theme.of(context);
```

Comment:
Are we guaranteed a Theme if it's a CupertinoApp?

Reply:
Yes Theme.of always returns a fallback theme at the least